ogcapi-draft/ogcapi-filter/src/main/java/de/ii/ogcapi/filter/app/FilterBuildingBlock.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2022 interactive instruments GmbH
+ *
+ * <p>This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy
+ * of the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package de.ii.ogcapi.filter.app;
+
+import de.ii.ogcapi.filter.domain.FilterConfiguration;
+import de.ii.ogcapi.foundation.domain.ExtensionConfiguration;
+import de.ii.ogcapi.foundation.domain.ApiBuildingBlock;
+import de.ii.ogcapi.filter.domain.ImmutableFilterConfiguration.Builder;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import com.github.azahnen.dagger.annotations.AutoBind;
+
+/**
+ * @title Modul "Filter / CQL" (FILTER)
```

Comment:
@CabukEge Always use the english title, so in this case `Filter - CQL (FILTER)`.